api/controllers/countries.js

```diff
@@ -72,7 +72,7 @@ function getCountryCriticalSites(req, res) {
   const query = `
     WITH csn_species_count AS (
       SELECT COUNT(*) AS csn_species, site_id
-      FROM csn_species_sites
+      FROM populations_sites
```

Comment:
I'm not sure if this csn_species_sites includes population information or just species-level information. Maybe we can have a look at it together. As species and populations are slightly different concepts.

I would also suggest keeping "critical" in the name, as the distinction at site level is between:

- critical sites;
- important biodiversity areas (ibas);

They are all sites but some are "IBA" and others are "Critical"

Reply:
The thinking here was that the `csn_species_sites` links the `populations_iba` and sites tables via `sites_id` and `wpepopid`, respectively.

Reply:
It links to populations, species and sites... Will rename to `critical_species_sites`